package.json

```diff
@@ -82,7 +82,7 @@
         "test": "BABEL_ENV=test NODE_ENV=test yarn jest",
         "test:e2e": "BROWSERSLIST_ENV=test npm-run-all -p -r start:examples cy:run",
         "test:e2e:open": "BROWSERSLIST_ENV=test npm-run-all -p -r start:examples cy:open",
-        "test:images": "yarn test -c scripts/jest/jest.config.js",
+        "test:images": "yarn test -c scripts/jest/jest.config.js --maxWorkers=4",
```

Comment:
Without this option, the full suite of visual tests would fail due to timeouts. Limiting the number of workers [significantly improves test execution time](https://jestjs.io/docs/en/troubleshooting#tests-are-extremely-slow-on-docker-andor-continuous-integration-ci-server). I also tried the other recommended option for expediting tests, `--runInBand`, but `--maxWorkers=4` consistently finished executing tests in less time (~15 seconds faster).